source/api/commands/defineSession.md

```diff
@@ -0,0 +1,40 @@
+---
+title: defineSession
+containerClass: experimental
+---
+
+{% note warning %}
+{% fa fa-warning orange %} **This is an experimental feature. In order to use it, you must set the {% url "`experimentalSessionSupport`" experiments %} configuration option to `true`.**
+{% endnote %}
+
+Defines a `Session`. A `Session` a collection of `cookies` and `localStorage` across all domains that may be restored to at any point during a test with the accompanying {% url `cy.useSession()` useSession %} command.
```

Comment:
I probably would not put `Session` in code ticks. It's just going to be complicated to keep up with and doesn't seem like it's strictly 'code' per se, more of a concept.